server/plugin/webhook.go

```diff
@@ -893,6 +894,68 @@ func (p *Plugin) handleCommentAuthorNotification(event *github.IssueCommentEvent
 	p.sendRefreshEvent(authorUserID)
 }
 
+func (p *Plugin) handleCommentAssigneeNotification(event *github.IssueCommentEvent) {
+	author := event.GetIssue().GetUser().GetLogin()
+	assignees := event.GetIssue().Assignees
+
+	for _, assignee := range assignees {
+		userID := p.getGitHubToUserIDMapping(assignee.GetLogin())
+		if userID == "" {
+			continue
+		}
+
+		if author == assignee.GetLogin() {
+			return
+		}
+		if event.Sender.GetLogin() == assignee.GetLogin() {
+			return
+		}
+
+		authorUserID := p.getGitHubToUserIDMapping(author)
+		if authorUserID == "" {
+			return
+		}
+
+		assigneeID := p.getGitHubToUserIDMapping(assignee.GetLogin())
+		if assigneeID == "" {
+			return
+		}
+
+		if event.GetRepo().GetPrivate() && !p.permissionToRepo(authorUserID, event.GetRepo().GetFullName()) {
+			return
+		}
+
+		splitURL := strings.Split(event.GetIssue().GetHTMLURL(), "/")
+		if len(splitURL) < 2 {
+			return
+		}
+
+		var templateName string
+		switch splitURL[len(splitURL)-2] {
+		case "pull":
+			templateName = "commentAuthorPullRequestNotification"
```

Comment:
Should there be different message? It's not really "your pull request" (https://github.com/mattermost/mattermost-plugin-github/blob/41f3bf52bd4b98e416aafccf96878aeec773da77/server/plugin/template.go#L279)

Reply:
yeah, used template of author. Any recommendations for template?

Reply:
No specific ones. I'm happy to review your proposal.